### ISSUE TITLE
Migration issue

### ISSUE THREAD

**Issue by frague59** [celery/django-celery-beat#137]
Hi, 

I've the following issue when upgrading celery-beat:

```
CommandError: Conflicting migrations detected; multiple leaf nodes in the migration graph: (0006_auto_20180210_1226, 0007_auto_20180219_1427 in django_celery_beat).
```
How can I fix this ?

Thanks for this great tool !

**Comment by frague59** [comment]
I'm using django 1.11.11 with :

```sh
$ pip list | grep celery
celery                             4.1.0      
django-celery                      3.2.2      
django-celery-beat                 1.1.1      
django-celery-results              1.0.1 
```
Thanks

**Comment by adriancarayol** [comment]
Same here, any solution @frague59?

**Comment by xeor** [comment]
Stange... I'm also here.. My issue was depending on `0006_auto_20180210_1226` in another apps migration-file..

It doesnt exist anymore.. Is it renamed? 0006 is not named `0006_auto_20180322_0932.py`

**Comment by auvipy** [comment]
that was a critical mistake, btw a new pr was merged for sorting the migrations out, 639ceb2d6eec0d9cf3771ad6025d791d622c85cc could you test the master?
